manifests/e/Elgato/CameraHub/1.3.0.596/Elgato.CameraHub.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Elgato.CameraHub
+PackageVersion: 1.3.0.596
+PackageLocale: en-US
+Publisher: Elgato Systems
+PublisherUrl: https://www.elgato.com/en
+PrivacyUrl: https://www.elgato.com/en/privacy-policy
+Author: Elgato Systems
+PackageName: Camera Hub

```

```suggestion
PackageName:  Elgato Camera Hub
```
This must match Control Panel.